hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/HBaseInterClusterReplicationEndpoint.java

```diff
@@ -113,6 +113,14 @@
   private boolean dropOnDeletedTables;
   private boolean isSerial = false;
 
+  protected Connection createConnection(Configuration conf) throws IOException {
+    return ConnectionFactory.createConnection(conf);
```

Comment:
This is wrong?  Should be ClusterConnectionFactory.createAsyncClusterConnection?

Reply:
Yeah, this was originally for a branch-2 based code. Had fixed it on the latest commit.